app/forms/declaration/consignor/ConsignorEoriNumber.scala

```diff
@@ -36,15 +36,14 @@ object ConsignorEoriNumber extends DeclarationPage {
 
   val mapping: Mapping[ConsignorEoriNumber] = Forms.mapping(
     eori -> mandatoryIfEqual(hasEori, YesNoAnswers.yes, Eori.mapping("declaration")),
-    hasEori -> optionalRadio("declaration.consignorEori.hasEori.empty", Seq(YesNoAnswers.yes, YesNoAnswers.no))
-      .transform[Option[String]](choice => Option(choice), choice => choice.getOrElse(""))
+    hasEori -> requiredRadio("declaration.consignorEori.hasEori.empty", Seq(YesNoAnswers.yes, YesNoAnswers.no))
```

Comment:
Nit: Inside `YesNoAnswer` object you have `allowedValues` that you could use here.

Reply:
Yup, previous implementation, but good point.

Changed